src/esp/gfx/RenderCamera.h

```diff
@@ -29,12 +29,10 @@ class RenderCamera : public Magnum::SceneGraph::AbstractFeature3D {
 
   // Overloads to avoid confusion
   scene::SceneNode& object() {
-    return static_cast<scene::SceneNode&>(
-        Magnum::SceneGraph::AbstractFeature3D::object());
+    return static_cast<scene::SceneNode&>(MagnumCamera::object());
   }
   const scene::SceneNode& object() const {
-    return static_cast<const scene::SceneNode&>(
-        Magnum::SceneGraph::AbstractFeature3D::object());
+    return static_cast<const scene::SceneNode&>(MagnumCamera::object());
   }
 
   void setProjectionMatrix(int width,
```

Comment:
We may want to consider returning reference to self (`RenderCamera&`) in both `setProjectionMatrix` overloads to stay consistent with `MagnumCamera::setProjectionMatrix`. Let me know your thoughts.

Reply:
Yes, correct. Let us do that, which is for method chaining so that you can call like object.methodA().methodB().methodC();